participant/src/lib.rs

```diff
@@ -51,13 +60,94 @@ pub fn request_inputs(input: &mut impl BufRead, logger: &mut dyn Logger) -> Resu
 
     input.read_line(&mut signing_share_input).unwrap();
 
-    let signing_share =
-        <[u8; 32]>::from_hex(signing_share_input.trim()).map_err(|_| Error::MalformedSigningKey)?;
+    let signing_share = SigningShare::from_bytes(
+        <[u8; 32]>::from_hex(signing_share_input.trim()).map_err(|_| Error::MalformedSigningKey)?,
+    )
+    .unwrap(); // TODO: Need the correct error from frost
+
+    // TODO: Is extra validation needed here for public_key and signing_share or will that be resolved when used in generating key_packages etc.? Need to check
+
+    logger.log("Your verifiable secret sharing commitment:".to_string());
```

Comment:
The trusted dealer demo prints `"Your commitment"` so users might not recognize that's the thing they need to input here. My suggestion is making the two equal (maybe changing the trusted dealer to match this?)

Reply:
Yes, good point.

Reply:
Done [9c7d886](https://github.com/ZcashFoundation/frost-zcash-demo/pull/35/commits/9c7d886956b1599f46711d1b05d69059f0aacb1c)